api/v1alpha08/sonataflow_types.go

```diff
@@ -657,6 +657,9 @@ type SonataFlowSpec struct {
 	// PodTemplate describes the deployment details of this SonataFlow instance.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="podTemplate"
 	PodTemplate PodTemplateSpec `json:"podTemplate,omitempty"`
+	// Persists service to a datasource of choice. Ephemeral by default.
+	// +optional
+	Persistence *PersistenceOptions `json:"persistence,omitempty"`
```

Comment:
@ricardozanini shouldn't the decision about persistency of the runtime be determined by the Platform?

Reply:
The idea is that this field can be used to override the platform's default persistence, or define one in case it does not exist. The platform's persistence field does not exist yet, it will be implemented in [KOGITO-9980](https://issues.redhat.com/browse/KOGITO-9980)

Reply:
Ok, that makes sense then.